### PR TITLE
doc(android) : getiing started and version

### PR DESCRIPTION
**Pull request by arthur740212**

### CODE REVIEW

File: docs/android/readme_android.md [hunk 1]
```diff
@@ -1,28 +1,67 @@
...
+
+
+# <div align="center">$\textcolor{deeppink}{\textsf{Android Getting Started}}$</div> <a name="getting-started" />
+---
+
+## Prerequisites
+
+
+* The standard <a href="/readme.md#prerequisites">Prerequisites</a> for all platforms.
+* Windows Build Support (IL2CPP)
```
Comment (andrew-hirata-playeveryware) [inline docs/android/readme_android.md]: Is windows build support actually required? That's somewhat shocking if it is true.

Comment (arthur740212) [reply docs/android/readme_android.md]: I thought it was. It was in the docs I written back when we started android
But I also question it now.

Comment (arthur740212) [reply docs/android/readme_android.md]: Should I remove it for now?

Comment (RedMazer) [reply docs/android/readme_android.md]: ill run things through a test process, so ill check this when i get there and make any needed changes

**Pull request merged**